src/syscheckd/src/run_check.c

```diff
@@ -128,12 +128,12 @@ void send_syscheck_msg(const cJSON *_msg) {
         return;
     }
 
-    static atomic_int_t n_msg_sent = ATOMIC_INT_INITIALIZER(0);
+    /*static atomic_int_t n_msg_sent = ATOMIC_INT_INITIALIZER(0);
 
     if (atomic_int_inc(&n_msg_sent) >= syscheck.max_eps) {
         sleep(1);
         atomic_int_set(&n_msg_sent, 0);
-    }
+    }*/
```

Comment:
is there any reason for these lines to be commented out?
I have checked and this functions are not used anymore, maybe we should remove them

Reply:
I understand the point, we could leave it for the topic

https://github.com/wazuh/wazuh/issues/10225